src/global_distrib/mod_global_distrib_utils.erl

```diff
@@ -145,12 +145,12 @@ create_ets(Names, Type) when is_list(Names) ->
 create_ets(Name, Type) ->
     Self = self(),
     Heir = case whereis(ejabberd_sup) of
```

Comment:
not the first time I see this pattern, usually in module `start` function.
Maybe we should have a dedicated process for keeping our ets tables.
Also, explicit `ets:give_away` is preferable, because the ets always would be owned by the same process.

Reply:
`mongoose_ets`? :) I'll create a task for this.